migrations/898-comm-cc-nonuser.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `comm_thread_cc`
+    MODIFY COLUMN `user_id` int(11) unsigned NULL,
+    ADD COLUMN `nonuser_email` varchar(255) NULL;
```

Comment:
yeah it could rebuild the cc list from the current mozilla_contact list each time.